README.md

```diff
@@ -12,47 +8,38 @@ Build on top of Plotly.js, React, and Flask, Dash ties modern UI elements like d
 [![LGTM Alerts](https://img.shields.io/lgtm/alerts/g/plotly/dash.svg)](https://lgtm.com/projects/g/plotly/dash/alerts)
 [![LGTM Grade](https://img.shields.io/lgtm/grade/python/g/plotly/dash.svg)](https://lgtm.com/projects/g/plotly/dash/context:python)
 
-[ONLINE DOCUMENTATION](https://dash.plot.ly/getting-started)
-
-[OFFLINE (PDF) DOCUMENTATION](https://github.com/plotly/dash-docs/blob/master/pdf-docs/Dash_User_Guide_and_Documentation.pdf)
 
-[DASH DOCS ON HEROKU](http://dash-docs.herokuapp.com/) (for corporate networks that cannot access plot.ly)
+#### *Dash is a Python framework for building analytical web applications. No JavaScript required*.
 
+Build on top of Plotly.js, React and Flask, Dash ties modern UI elements like dropdowns, sliders, and graphs directly to your analytical python code. Read our tutorial proudly crafted ❤️ by dash itself.
 
-Here’s a 43-line example of a Dash App that ties a Dropdown to a D3.js Plotly Graph.
-As the user selects a value in the Dropdown, the application code dynamically
-exports data from Google Finance into a Pandas DataFrame. This app was written in just 43 lines of code ([view the source](https://gist.github.com/chriddyp/3d2454905d8f01886d651f207e2419f0)).
+- [User Guide](https://dash.plot.ly/getting-started)
 
-![Sample Dash App](https://user-images.githubusercontent.com/1280389/30086128-9bb4a28e-9267-11e7-8fe4-bbac7d53f2b0.gif)
+- [Offline (PDF) Documentation](https://github.com/plotly/dash-docs/blob/master/pdf-docs/Dash_User_Guide_and_Documentation.pdf)
 
-Dash app code is declarative and reactive, which makes it easy to build complex apps that contain many interactive elements. Here’s an example with 5 inputs, 3 outputs, and cross filtering. This app was composed in just 160 lines of code, all of which were Python.
+- [Dash Docs on Heroku](http://dash-docs.herokuapp.com/) (for corporate network that cannot access plot.ly)
 
-![crossfiltering dash app](https://user-images.githubusercontent.com/1280389/30086123-97c58bde-9267-11e7-98a0-7f626de5199a.gif)
 
-Dash uses [Plotly.js](https://github.com/plotly/plotly.js) for charting. Over 35 chart types are supported, including maps.
+### App Samples
 
- ![Dash app with Mapbox map showing walmart store openings](https://user-images.githubusercontent.com/1280389/30086299-768509d0-9268-11e7-8e6b-626ac9ca512c.gif)
- 
-Dash isn't just for dashboards. You have full control over the look and feel of your applications. Here's a Dash app that's styled to look like a PDF report.
-
-![goldman sachs report](https://user-images.githubusercontent.com/1280389/30086373-d076a372-9268-11e7-99df-d84fa69f3e20.gif)
+| App | Description |
+|--- | :---: |
```

Comment:
I'd remove the header row, doesn't add anything useful. Also, might be just personal preference but I'd left-align the text rather than centering it. I do like the table format for listing the apps!

Reply:
well, github markdown doesn't support table without header row, change it back
![Screen Shot 2019-05-14 at 11 15 40 AM](https://user-images.githubusercontent.com/1394467/57709880-d62d0980-7639-11e9-9cd1-78149d0b6c0a.png)